integrations/quickstart-Other.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: "Quickstart Other CIs"
+description: "Trigger Octomind in your pipeline via cURL"
+icon: "person-digging"
+---
+
+## Using the REST endpoint
+
+For other platforms like bitbucket or gitlab you can consult <a href="execution-without-ci">the documentation for triggering a report via the REST endpoint</a>.
```

Comment:
don't use an a element, the link will break on deployment due to our deployment with url rewrite, use a markdown one with [] ()